test/release/examples/PREDIFF

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-sort $2 > $2.prediff.tmp
+sed -e 's/\r$//' -e '/^salloc:/d' < $2 | sort > $2.prediff.tmp
```

Comment:
I tried resurrecting that global prediff. It did not actually filter out salloc outputs, which is one thing I needed. It added a blank line to the output, now and then.  Finally, it is a large perl program, doing a lot of stuff I did not need done. So I decided to just use the simple sed command I had before, except now in a global PREDIFF instead of the per-directory PREDIFF.